.github/workflows/base-image-check.yml

```
@@ -124,6 +126,51 @@ jobs:
             echo "\`${TAG}\` is the latest tag (checked up to \`${MAJOR}.$((MINOR + 5))\`)." >> $GITHUB_STEP_SUMMARY
           fi
 
+      - name: Check for newer UBI major
+        id: ubi
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2001:style:21:21: See if you can use ${variable//search/replace} instead

(shellcheck)